docs/resources/advanced_cluster.md

```diff
@@ -598,7 +596,7 @@ If you are upgrading a replica set to a sharded cluster, you cannot increase the
 
 ~> **IMPORTANT:** If `disk_gb_enabled` is true, then Atlas will automatically scale disk size up and down.
 This will cause the value of `replication_specs.#.region_config.#.(analytics_specs|electable_specs|read_only_specs).disk_size_gb` returned to potentially be different than what is specified in the Terraform config and if one then applies a plan, not noting this, Terraform will scale the cluster disk size back to the original `disk_size_gb` value.
-To prevent this a lifecycle customization should be used, i.e.:  
+To prevent this a lifecycle customization should be used:  
```

Comment:
Rewrite as: "To prevent disk scaling, ..."

In the prev line 598, where it says "if one uses", change it to "if you use, and then apply".